tests/python_client/testcases/test_index.py

```diff
@@ -214,6 +214,19 @@ def test_index_create_with_different_indexes(self):
         assert len(collection_w.indexes) == 1
         assert collection_w.indexes[0].params["index_type"] == default_index_params["index_type"]
 
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.xfail(reason="issue 19972")
+    def test_index_create_indexes_for_different_fields(self):
+        """
+        target: Test create indexes for different fields
+        method: create two different indexes
```

Comment:
Here the description should address "create two different indexes with default index name".